services/amo/amo-users.service.js

```diff
@@ -0,0 +1,46 @@
+'use strict'
+
+const { metric } = require('../../lib/text-formatters')
+const { BaseAmoService, keywords } = require('./amo-base')
+
+module.exports = class AmoUsers extends BaseAmoService {
+  static get category() {
+    return 'downloads'
+  }
+
+  static get route() {
+    return {
+      base: 'amo/users',
+      pattern: ':addonId',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Mozilla Add-on',
+        namedParams: { addonId: 'dustman' },
+        staticPreview: this.render({ users: 750 }),
+        keywords,
+      },
+    ]
+  }
+
+  static get defaultBadgeData() {
+    return {
+      label: 'users',
+    }
+  }
+
+  static render({ users }) {
+    return {
+      message: metric(users),
+      color: 'brightgreen',
```

Comment:
I know this is consistent with the legacy service, but if the color will always be the same, would informational/blue make more sense than success/brightgreen? Or perhaps a scale?